chart/values.yaml

```diff
@@ -1297,6 +1297,13 @@ redis:
     # Annotations to add to worker kubernetes service account.
     annotations: {}
 
+  uid: 999
```

Comment:
Using 999 as the default does change the behavior as it runs as `redis` not `root`, but that worked for me even if the redis db was owned as `root`. If we want to be extra safe though, we might consider defaulting to 0 instead. Thoughts?